client/landing/stepper/declarative-flow/internals/steps-repository/readymade-template-generate-content/components/readymade-template-preview/style.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../../style";
+
+$font-family: "SF Pro Text", $sans;
+
+.readymade-template-preview {
+	width: 50%;
```

Comment:
We need to add some responsiveness for mobile viewports

<img width="509" alt="Screenshot 2024-08-02 at 13 44 26" src="https://github.com/user-attachments/assets/3f7a688b-7422-407c-81fd-cd294900cf8e">

Reply:
Added an item to the PT issue for this.